frontend/openchat-agent/src/services/canisterAgent/msgpack.ts

```diff
@@ -90,22 +86,21 @@ export abstract class MsgpackCanisterAgent extends CanisterAgent {
         try {
             const payload = MsgpackCanisterAgent.prepareMsgpackArgs(args, requestValidator);
 
-            const { requestId, response } = await this.sendRequestToCanister(() =>
-                this.agent.call(this.canisterId, {
-                    methodName: methodName + "_msgpack",
-                    arg: payload,
-                    callSync: onRequestAccepted === undefined,
-                }),
-            );
+            const { requestId, response } = await this.agent.call(this.canisterId, {
+                methodName: methodName + "_msgpack",
+                arg: payload,
+                callSync: onRequestAccepted === undefined,
+            });
             const canisterId = Principal.fromText(this.canisterId);
 
-            if (response.ok && response.body?.certificate) {
-                const certTime = this.agent.replicaTime;
+            if (response.ok && (response.body as v3ResponseBody)?.certificate) {
+                const cert = (response.body as v3ResponseBody).certificate;
+                // const certTime = this.agent.replicaTime;
                 const certificate = await Certificate.create({
-                    certificate: bufFromBufLike(response.body?.certificate),
+                    certificate: bufFromBufLike(cert),
                     rootKey: this.agent.rootKey,
                     canisterId: Principal.from(canisterId),
-                    certTime,
+                    blsVerify: undefined,
```

Comment:
I have no idea what this `blsVerify` and I don't know how to get it. The agent code (that I'm pretty sure this is based on) seems to get it out of thin air and I don't know if it matters.

Reply:
Setting it undefined looks fine, when we call `createActor` we don't set it, so in all our candid use cases it will also be undefined.